### ISSUE TITLE
GARCH model implementation

### ISSUE THREAD

**Issue by christianadriano** [hpi-sam/rl-4-self-repair#22]
Rachel, could you please also add the GARCH model to the environment as a function and run the t-tests on the resulting time series?

Follows example implementation:
https://colab.research.google.com/drive/1NUP5kmQzNOQjj1GVG6TtiIN4fmi_qNyH#scrollTo=tJY1NUYhiiuU

**Comment by brrrachel** [comment]
What are the start and end values for a series for <component,failure>?

**Comment by christianadriano** [comment]
The start is could be the minimum value of the original series, e.g., Min_accuracy_increase(<C1,F2>)
You might also need the initial variance, which could be the variance of the original series.

Do we need to define the end? I think we only need to define the coefficients for the Garch model.

**Comment by brrrachel** [comment]
No I do not understand the variables of the function - how is this influenced by <component,failure>?

**Comment by christianadriano** [comment]
In this code here https://colab.research.google.com/drive/1NUP5kmQzNOQjj1GVG6TtiIN4fmi_qNyH#scrollTo=tJY1NUYhiiuU

I believe that we only need to change the value of t[0]. All the rest can stay the same. t[0] can be set be the minimum value of accuracy_increase of <component,failure> pair.

**Comment by brrrachel** [comment]
Okay I will set t[0].
But I would use the mean value of the transformed and shifted pair ?!?

**Comment by christianadriano** [comment]
Yes. Since the series produced by the Garch model fluctuates around a mean, it would make sense to use the mean of the transformed_shifted pair.

**Comment by brrrachel** [comment]
Running the ttest, which ordering to take?
If I use the ordering from before working on the data, I get different results as if I do this based on the ordering retrieved from the new data.
